examples/car_control/run.py

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+# Before running this file, rename user_config.yaml.example -> user_config.yaml and modify it with appropriate paths for your system.
+
+import argparse
+
+import numpy as np
+import os
+import spear
+import time
+
+common_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), "..", "common"))
+import sys
+
+sys.path.append(common_dir)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--actions_file", default=os.path.realpath(os.path.join(os.path.dirname(__file__), "actions.csv")))
+    parser.add_argument("--bp_class_reference", default="/Game/VehicleTemplate/Blueprints/OffroadCar/OffroadCar_Pawn.OffroadCar_Pawn_C")
+    args = parser.parse_args()
+
+    np.set_printoptions(linewidth=200)
+
+    # load config
+    config = spear.get_config(
+        user_config_files=[
+            os.path.realpath(os.path.join(common_dir, "default_config.common.yaml")),
+            os.path.realpath(os.path.join(os.path.dirname(__file__), "user_config.yaml"))])
+
+    spear.configure_system(config)
+    instance = spear.Instance(config)
+
+    instance.engine_service.begin_tick()
+
+    gameplay_statics_class = instance.unreal_service.get_static_class(class_name="UGameplayStatics")
```

Comment:
Group these functions more logically. Do all the `find_function` ops together, do all the `load_class` ops together, etc.

Reply:
fixed